docs/data/material/components/accordion/ControlledAccordions.js

```diff
@@ -3,8 +3,29 @@ import Accordion from '@mui/material/Accordion';
 import AccordionDetails from '@mui/material/AccordionDetails';
 import AccordionSummary from '@mui/material/AccordionSummary';
 import Typography from '@mui/material/Typography';
+import { styled } from '@mui/material/styles';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 
+const SummaryText = styled('span')(({ theme }) => ({
+  margin: 0,
+  textAlign: 'left',
```

Comment:
Is this needed now that `AccordionSummaryContent` has `textAlign: start`?

Reply:
Not needed. Removed it.